.env

```diff
@@ -0,0 +1,3 @@
+DB_USER = market
```

Comment:
Good job, but '.env' needs to be added to 'gitignore', otherwise it turns out - we hide the data in the env file, and the file itself is visible to everyone.